core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/PurchasesDelegate.ios.kt

```diff
@@ -15,8 +15,9 @@ import platform.darwin.NSObject
 internal fun RCPurchasesDelegateProtocol.toPurchasesDelegate(): PurchasesDelegate =
     (this as PurchasesDelegateWrapper).wrapped
 
-internal fun PurchasesDelegate.toRcPurchasesDelegate(): RCPurchasesDelegateProtocol =
-    PurchasesDelegateWrapper(this)
+internal fun PurchasesDelegate?.toRcPurchasesDelegate(): RCPurchasesDelegateProtocol? =
+    this?.let { PurchasesDelegateWrapper(it) }
+        .also { PurchasesDelegateStrongReference.delegate = it }
```

Comment:
This is changed so that when the `delegate` is set to null, the strong reference is cleared.